### ISSUE TITLE
avoid eval

### ISSUE THREAD

**Issue by ludans** [forzagreen/n2words#3]
var num = eval(`new Num2Word_${lang}()`);

causes issues when packaging with rollup: treeshake algorithm does not bundle the functions for each language, as they are not explicitely classed

would it be possible to just avoid eval here?

**Comment by forzagreen** [comment]
Yes. Thanks for reporting. I will check it.

**Comment by ludans** [comment]
I temporarly patched like this:
```
let num;
        if (lang === 'EN') {
          num = new Num2Word_EN();
        } else if (lang === 'FR') {
          num = new Num2Word_FR();
        } else if (lang === 'DE') {
          num = new Num2Word_DE();
        } else if (lang === 'IT') {
          num = new Num2Word_IT();
        }
```

**Comment by forzagreen** [comment]
Fixed in `v1.3.2`

**Comment by ludans** [comment]
thanks!
